src/store/lokapi.ts

```diff
@@ -63,20 +63,28 @@ export function lokapiStoreFactory(lokApiService: any) {
       },
       async fetchAccounts({commit}:any) {
         commit("setAccountsLoading", true)
-        const { virtualAccountTree, allMoneyAccounts } = await lokApiService.buildVirtualAccountTree()
-        commit("setAccounts", { virtualAccountTree, allMoneyAccounts })
+        try {
+          const { virtualAccountTree, allMoneyAccounts } = await lokApiService.buildVirtualAccountTree()
+          commit("setAccounts", { virtualAccountTree, allMoneyAccounts })
+        } catch (e:any) {
+          console.error('Error fetching wallets', e.message, "\nDebug: " + JSON.stringify(e))
+        }
```

Comment:
You are hiding the error to the UI and prevent it to take the choice to display or not message by catching the exception and display something or just ignore it. I don't think this is the way to go.
Inner layer (like the store) should let these exception bubble up to let the choice to the UI of the course of action.
You can of course add some behavior on the go here like removing `setAccountsLoading` to false, just don't forget to throw the err after.

Reply:
? I think there is a misunderstanding here:
One commit is in charge of catching the errors and make the accounts loading-indicator at least stop. Which is an improvement over the previous state of things which rendered the UI loading indefinitely.
The next and last commit of this PR is responsible of adding feedback to the user, which again is an improvement over the previous commit. I could have done it sqashed. I made this on purpose. First fix something broken. Then add some functionality.

Then again, as some replies to your comments below state, you are missing that the `fetchAccounts` action is not called from a component but from the store itself, so this is the place where the exception should be catched and action taken. As my replies below also state, I'd really be interested to know how you would do it, as I may be missing something.

Reply:
Squashed commits. Changed.